tools/isobuild/resolver.ts

```diff
@@ -77,11 +79,13 @@ export default class Resolver {
     sourceRoot,
     targetArch,
     extensions = [".js", ".json"],
+    npmDependencies,
     nodeModulesPaths = [],
   }: ResolverOptions) {
     this.sourceRoot = sourceRoot;
     this.extensions = extensions;
     this.targetArch = targetArch;
+    this.npmDependencies = Object.keys(npmDependencies || {});
```

Comment:
npmDependencies can be undefined, but also null. So we can't use a default function parameter in the constructor signature.